docs/pguidance/agent-install-packages.md

```diff
@@ -0,0 +1,142 @@
+* Kernel version 2.6 or higher
+* CAP_DAC_READ_SEARCH and CAP_SYS_PTRACE capabilities
+* APT or similar package tools. These instructions show you how to install the package using `apt-get`.
+* Internet access. If necessary, set up proxies to allow your package tools to access the Internet.
+* Terminal or a similar command-line interface application
+* Permission to run `curl` and `sudo`
+
+### Install the SignalFx Smart Agent with Debian
+
+1. Remove collector services such as `collectd`.
+
+2. Remove third-party instrumentation and agent software.
+
+> Don't use automatic instrumentation or instrumentation agents from
+> other vendors when you're using SignalFx instrumentation. The results
+> are unpredictable, but your instrumentation may break and your
+> application may crash.
+
+3. Download the GNU Privacy Guard (**GnuPG**) security key for the Debian package
+   by running the following command:
+
+   ```
+   curl -sSL https://splunk.jfrog.io/splunk/signalfx-agent-deb/splunk-B3CD4420.gpg > /etc/apt/trusted.gpg.d/splunk.gpg
+   ```
+
+4. Add an entry for the SignalFx Smart Agent package to Debian by running the following command:
+
+   ```
+   echo 'deb https://splunk.jfrog.io/splunk/signalfx-agent-deb release main' > /etc/apt/sources.list.d/signalfx-agent.list
+   ```
+
+5. Update the Debian package lists with the SignalFx Smart Agent package information
+   by running the following command:
+
+   ```
+   apt-get update
+   ```
+
+6. Install the Agent by running the following command:
+
+   ```
+   apt-get install -y signalfx-agent
+   ```
+
+### Configure Debian installation
+
+Set the Smart Agent configuration options in the configuration YAML file. To learn more,
+see [Agent Configuration](../config-schema.md).
+
+### Verify Debian installation
+
+See [Verify the Smart Agent](#verify-the-smart-agent).
+
+## Install using RPM
+
+This section describes how to install the Smart Agent on hosts that
+run Linux and use the RPM package manager.
+
+### Prerequisites for RPM
+
+* *nix distribution that supports RPM, such as CentOS or Oracle Linux
+* Kernel version 2.6.32 or higher
+* CAP_DAC_READ_SEARCH and CAP_SYS_PTRACE capabilities
+* `yum` or similar package tools. These instructions show you how to install the package using `yum`.
+* Write access to `/etc/yum.repos.d/`
+* Internet access. If necessary, set up proxies to allow your package tools to access the Internet.
+* Terminal or a similar command-line interface application
+
+### Install with RPM
+
+1. Remove collector services such as `collectd`.
+2. Remove third-party instrumentation and agent software.
+
+> Don't use automatic instrumentation or instrumentation agents from
+> other vendors when you're using SignalFx instrumentation. The results
+> are unpredictable, and your instrumentation might break and your
+> application might crash.
+
+3. In the `/etc/yum.repos.d/` directory, edit a new file named signalfx-agent.repo.
```

Comment:
Do they need to create the new file before they can edit the new file? Where does the new file come from?